util.go

```diff
@@ -12,6 +14,17 @@ import (
 	"github.com/pion/transport/vnet"
 )
 
+func init() { // nolint:gochecknoinits
```

Comment:
To call this in the first use of random, any other call of `randSeq` and `generateRandString` should wait until calling `Seed()`, otherwise these parallel calls may return random generated by the default seed.
It makes code complex and may cause future bug, so I believe initializing in `init()` is the best solution.